packages/core/helper-plugin/lib/src/components/DateTimePicker/index.js

```diff
@@ -1,213 +1,17 @@
-import React, { useEffect, useState } from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { Stack } from '@strapi/design-system/Stack';
-import { DatePicker } from '@strapi/design-system/DatePicker';
-import { TimePicker } from '@strapi/design-system/TimePicker';
-import { Field, FieldHint, FieldLabel, FieldError } from '@strapi/design-system/Field';
-import { Flex } from '@strapi/design-system/Flex';
-import { Box } from '@strapi/design-system/Box';
-import { parseDate } from './parseDate';
```

Comment:
Can we delete this file now (I don't see it showing up in the diff)?

Reply:
yes removed and also removed the test